index.html

```diff
@@ -133,18 +133,17 @@ <h4 class="wow fadeIn" data-wow-delay=".6s" lang="en">Get the most complete Linu
             <span class="opensuse-countdown__hours opensuse-countdown__hours__singular" lang="en">hour</span>
           </div>
           <div class="opensuse-countdown__call-to-action">
-            <span lang="en">for openSUSE Conference</span>
+            <span lang="en">for Leap 42.3</span>
           </div>
         </a>
       </div>
       <div class="opensuse-countdown__released hidden">
-        <a href="https://media.ccc.de/c/osc17" target="_blank">
+        <a href="https://www.youtube.com/user/opensusetv" target="_blank">
```

Comment:
Did they require a link to youtube? normally it is to the download button. Check in the history of this file